requirements.dso.txt

```diff
@@ -1,3 +1,4 @@
+psycopg[binary]
 psycopg2-binary
```

Comment:
I am curious: why do we need both psycopg[binary] and psycopg2-binary?

Reply:
Like written in 0b03230397735973f159b95b98fa7f58b9064436, SQLAlchemy (ORM) still relies on `psycopg2`, while our None-ORM approach is implemented in `psycopg3` (packages is called `psycopg`)...

Reply:
understood. as discussed earlier, please add a _brief_ comment (inline, into this file, after the respective line) stating this (like: # required by sqlalchemy, or sth the like)